.github/workflows/deno.yml

```
@@ -11,6 +11,8 @@ name: grammY
 on:
     push:
         branches: ["*"]
+    pull_request:
+        branches: ["*"]
```

This line can be omitted as its (as far as I recall) the same

```suggestion
        branches: ["*"]
```